lib/graphql/base_type.rb

```diff
@@ -24,6 +34,24 @@ def initialize_copy(other)
     # @return [String, nil] a description for this type
     attr_accessor :description
 
+    # @return [Boolean] Is this type a predefined introspection type?
+    def introspection?
```

Comment:
We may be able to change this https://github.com/rmosolgo/graphql-ruby/blob/895d8b7c198d3db6d290bcc95f37e3767ab6b5f5/lib/graphql/schema/validation.rb#L181 now that we have `introspection?`.